PyTado/interface.py

```diff
@@ -343,41 +343,24 @@ def resetZoneOverlay(self, zone):
         data = self._apiCall(cmd, "DELETE", {}, True)
         return data
 
-    def setZoneOverlay(self, zone, overlayMode, setTemp=None, duration=None, deviceType='HEATING', power="ON", mode=None):
+    def setZoneOverlay(self, zone, overlayMode, setTemp=None, duration=None, deviceType='HEATING', power="ON", mode=None, fanSpeed=None):
         """set current overlay for a zone"""
         # pylint: disable=C0103
 
         cmd = 'zones/%i/overlay' % zone
 
         post_data = {
-            "setting" : {},
-            "termination" : {}
+            "setting": {"type": deviceType, "power": power},
+            "termination": {"type": overlayMode},
```

Comment:
the change in `setting` seems ok, but can we make sure, that setting `termination.type` always to `overlayMode` wouldn't break anything?